reclaimer/sound/ogg_new.py

```diff
@@ -0,0 +1,82 @@
+import struct
+from Encoder import OGGEncoder
+from supyr_struct.defs.audio.wav import wav_def
+
+"""
+    OGG Encoder:
+        Creation: 06/15/2020
+        Last Edit: 06/18/2020 (Zatarita)
+"""
```

Comment:
This should be the very first thing in the file if you want it to function as a docstring.

These docstrings can be put in as the very first declaration inside of functions, classes, etc so that when we generate documentation it's associated with the proper function/class. If it's put anywhere else it doesn't actually work this way.